packages/agreement-process/src/services/agreementContractBuilder.ts

```diff
@@ -201,6 +209,31 @@ const getPdfPayload = async (
           : undefined,
       };
     }),
+    delegationId: delegationData?.delegation.id,
+    delegatorName: delegationData?.delegator.name,
+    delegatorIpaCode: delegationData && getIpaCode(delegationData?.delegator),
+    delegateName: delegationData?.delegate.name,
+    delegateIpaCode: delegationData && getIpaCode(delegationData?.delegate),
```

Comment:
Could you add a `producer` prefix to all these delegation/delegator/delegate (and also in the `AgreementContractPDFPayload` model and in the PDF template where these vars are used)

Reply:
Solved [here](https://github.com/pagopa/interop-be-monorepo/pull/1175/commits/c375c2664bcd263fe62720448352b58d6334daa0)